modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,33 @@
+"""
+Modular Exponential
+"""
+
+def mod_exponent(base, power, mod):
+    """
```

Comment:
What u are saying, i was not able to understand, as this follows the codingstyle guidelines also. It is according to PEP8 guidelines. for reference [https://www.python.org/dev/peps/pep-0257/](https://www.python.org/dev/peps/pep-0257/)